changelog.d/20240702_175644_30907815+rjmello_exec_cmd_sc_34376.rst

```diff
@@ -0,0 +1,11 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Added the ``globus-compute-endpoint python-exec`` command to run Python modules as scripts
```

Comment:
This isn't exactly a user-facing change. It might be better to put this under Changed as `we no longer rely on PATH to resolve ... instead we use a new mechanism ... for more reliable worker starts for multi user endpoint users`. @rjmello what do you think?

Reply:
All good now :)